.github/workflows/label-triggered.yml

```
@@ -39,6 +39,30 @@ jobs:
         env:
           INVITE_TOKEN: ${{ secrets.RELEASEDRAFTER_PAT }}
 
+  add-comment-issue-valid:
+    if: github.event.label.name == 'issue/valid'
+    runs-on: ubuntu-latest
+    steps:
+      - name: Add comment
+        uses: peter-evans/create-or-update-comment@5f728c3dae25f329afbe34ee4d08eef25569d79f
+        with:
+          issue-number: ${{ github.event.issue.number }}
+          body: |
+              Thanks for opening this issue.It is a verified and valid issue you can take action in accordance
+              to description offered in the issue.
+
+  add-comment-help-wanted:
+    if: github.event.label.name == 'help-wanted'
+    runs-on: ubuntu-latest
+    steps:
+      - name: Add comment
+        uses: peter-evans/create-or-update-comment@5f728c3dae25f329afbe34ee4d08eef25569d79f
+        with:
+          issue-number: ${{ github.event.issue.number }}
+          body: |
+              Thank you for opening this issue extra attention is needed for this issue.
```

To increase readability I suggest restructuring this sentence to read as follows:
```suggestion
             Thank you for opening this issue. It requires special attention, and we appreciate your contribution.
```